src/components/MassEditDialog.vue

```diff
@@ -457,7 +457,7 @@ export default {
           album_id: t.album_id,
           review_comment: this.clearReviewComments ? null : t.review_comment,
           genre_ids: t.genre_ids,
```

Comment:
I guess this should also happen for `genre_ids`?

Reply:
true, fixed